### DISCUSSION TITLE
Finding Maintainers for Github Project

### DISCUSSION THREAD

**Discussion by whgibbo** [community/community#77095] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi,
I'm trying to find the maintainer to a Github Repo, to contribute  to the project.. But it seems to have died...

Is there a process in place for:

1. Getting a list of the maintainers / Owner to contact them directly ?
2. Being added to the list of Maintainers ?
3. Last resort, I guess I could fork the repo and maintain myself.

Thoughts?

**Comment by MantasTech** [comment, marked as answer]
Hi whgibbo!

My best advice would be to contribute to active projects or collab with friends.
However, to answer your question:

1. Some projects include  Discord/Telegram/Twitter. Also make sure to check for CONTRIBUTE.md if it is included in the project.
2. You can try to look profiles of the contributors and again, search for their socials like Linkedin/Discord and contact them directly.
3. Deffinetly, you can fork and make the changes you want just for you!

Hope that helped!
